eval/eval.py

```diff
@@ -158,7 +158,7 @@
 CatResultDict = Dict[str, Union[int, float, str]]
 
 settings = Settings()
-settings.read("../src/reynir_correct/config/GreynirCorrect.conf")
+settings.read("config/GreynirCorrect.conf")
```

Comment:
Better: `os.path.join("config", "GreynirCorrect.conf")` (works on Windows, with backslashes instead of forward slashes)

Reply:
...and is the current working directory guaranteed to be correct here?

Reply:
...but perhaps this is handled in settings.read(), and in that case, never mind.

Reply:
Changed, and yes, it's handled by settings.read()